tools/functional-tester/etcd-tester/key_stresser.go

```diff
@@ -202,6 +204,64 @@ func newStressPut(kvc pb.KVClient, keySuffixRange, keySize int) stressFunc {
 	}
 }
 
+func newStressTxn(kvc pb.KVClient, keyTxnSuffixRange int) stressFunc {
+	keys := make([]string, keyTxnSuffixRange)
+	for i := range keys {
+		keys[i] = fmt.Sprintf("/k%03d", i)
+	}
+	for i, k := range keys {
+		_, err := kvc.Put(context.Background(), &pb.PutRequest{
+			Key:   []byte(k),
+			Value: []byte{byte(i)},
+		}, grpc.FailFast(false))
+		if err != nil {
+			plog.Fatalf("newStressTxn error (%v)", err)
+		}
+	}
+
+	return func(ctx context.Context) (error, int64) {
+		n := rand.Intn(keyTxnSuffixRange)
```

Comment:
The number of keys selected shouldn't be coupled to the number of keys in the working set. Half the time it'll be checking the state of half the keys or more and only one key has to be present to delete everything. Probably should set it to some small number of ops like `n := rand.Intn(64)`  and the user can configure the conflict rate by adjusting the txn suffix range.